source/docs/articles/wordpress/site-networks/managing.md

```diff
@@ -0,0 +1,130 @@
+
+Because you just have one site on your Multisite Network, you can perform the search and replace in one operation:
+
+```bash
+terminus wp --site=handbuilt-site-network --env=test search-replace dev-handbuilt-site-network.pantheon.io test-handbuilt-site-network.pantheon.io --url=dev-handbuilt-site-network.pantheon.io --network
+```
+
+**Pro Tip**: Include the --dry-run flag to get a preview of the changes without destructively transforming the database.
+
+Before you run the command, let’s review each argument:
+
+`terminus wp` tells Terminus to perform a WP-CLI command
+`--site=handbuilt-site-network --env=test` are associative arguments interpreted by Terminus to know which site and which environment to run the WP-CLI command on
+`search-replace dev-handbuilt-site-network.pantheon.io test-handbuilt-site-network.pantheon.io` are three positional arguments interpreted by WP-CLI. WP-CLI interprets them as “run the search and replace command; use `dev-handbuilt-site-network.pantheon.io` as the string to search for; use `test-handbuilt-site-network.pantheon.io` as the value to replace it with.
+`--url=dev-handbuilt-site-network.pantheon.io` is an associative argument to set the context with which to load WordPress. Because the database still has the Dev URL stored, WordPress thinks it should be the Dev domain until the search and replace has been performed
+`--network` is a flag to perform the search and replace across the entire network
+
+Ready to commit? Run the command! When you do, you’ll see output similar to this:
+
+[tk sample output from search-replace execution]
+
+Now that you’ve performed the search and replace on your database, WordPress will load in your Test environment.
+
+### Advanced Search and Replace on WordPress Multisite with Multiple Sites
+
+Once your WordPress Multisite environment is established, you’ll likely add multiple sites to it. When you add multiple sites to your WordPress Multisite environment, performing the search and replace when copying the database between environments becomes a bit more involved.
+
+As an example, consider a WordPress Multisite environment in Live that you’d like to copy to Test. This Multisite environment has three sites, two of which have custom mapped domains.
+
+In Live, the primary domain for WordPress is pantheon.io. This first site’s dashboard lives at pantheon.io/wp-admin/. The second and third sites on the network have the slugs `second-site` and `third-site`. This means their dashboards live at `second-site.pantheon.io/wp-admin/` and `third-site.pantheon.io/wp-admin/`. However, each site has a mapped domain for the frontend — the second site is served at `second-site.com` and the third site is exposed at `third-site.com`.
```

Comment:
Do these instructions differ for network sites configured as subdirectories vs subdomains?